mods/tuxemon/db/npc/spyder_omnichannel_enforcertwo.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
this file is fine...just added here for visibility.  Please add your name onto the CREDITS file for the sprites that you have made, along with the license they are released under.  If unsure of the license, then find the license of the original items that you modified.